packages/presets/swc-plugin/tests/fixtures/simple-uppercase-operation-name.other-dir.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */ //@ts-ignore
-import { CFragmentDoc } from "../graphql";
-import { BDocument } from "../graphql";
-import { ADocument } from "../graphql";
+import { CFragmentDoc } from "./../graphql";
+import { BDocument } from "./../graphql";
+import { ADocument } from "./../graphql";
```

Comment:
@YassinEldeeb The `./` is kind of redundant as `./../graphql` is the same as `../graphql` 🤔

Reply:
we can improve it in a follow-up PR